packages/connect/src/types/api/cardanoComposeTransaction.ts

```diff
@@ -54,6 +72,120 @@ export type CardanoComposeTransactionParams = {
+    }),
+    feeLevels: Type.Optional(
+        Type.Array(
+            Type.Object({
+                feePerUnit: Type.Optional(Type.String()),
+            }),
+        ),
+    ),
+    outputs: Type.Optional(
+        Type.Array(
+            Type.Intersect(
+                [
+                    Type.Object(
+                        {
+                            isChange: Type.Optional(Type.Boolean()),
+                            assets: Type.Array(
+                                Type.Object({
+                                    unit: Type.String(),
+                                    quantity: Type.String(),
+                                }),
+                            ),
+                        },
+                        { $id: 'BaseOutput' },
+                    ),
+                    Type.Union([
+                        Type.Object(
+                            {
+                                address: Type.String(),
+                                amount: Type.String(),
+                                setMax: Type.Optional(Type.Literal(false)),
+                            },
+                            { $id: 'ExternalOutput' },
+                        ),
+                        Type.Object(
+                            {
+                                address: Type.Optional(Type.String()),
+                                amount: Type.Optional(Type.String()),
+                                setMax: Type.Boolean(),
+                            },
+                            { $id: 'ExternalOutputIncomplete' },
+                        ),
+                    ]),
+                ],
+                { $id: 'UserOutput' },
+            ),
+        ),
+    ),
+    certificates: Type.Optional(Type.Array(CardanoCertificate)),
+    withdrawals: Type.Optional(
+        Type.Array(
+            Type.Object(
+                {
+                    stakeAddress: Type.String(),
+                    amount: Type.String(),
+                },
+                { $id: 'Withdrawal' },
+            ),
+        ),
+    ),
+    changeAddress: Type.Object({
+        address: Type.String(),
+        path: Type.String(),
+    }),
+    addressParameters: Type.Object(
+        {
+            addressType: PROTO.EnumCardanoAddressType,
+            path: DerivationPath,
+            stakingPath: Type.Optional(DerivationPath),
+            stakingKeyHash: Type.Optional(Type.String()),
+            certificatePointer: Type.Optional(CardanoCertificatePointer),
+        },
+        { $id: 'CardanoAddressParameters' },
+    ),
+    testnet: Type.Optional(Type.Boolean()),
+});
+
+// TS - Assert type of CardanoComposeTransactionParams is compatible with CardanoComposeTransactionParamsSchema
+const _params: CardanoComposeTransactionParamsSchema = {} as CardanoComposeTransactionParams;
+const _paramsOld: CardanoComposeTransactionParams = {} as CardanoComposeTransactionParamsSchema;
+// eslint-disable-next-line
+[_params, _paramsOld];
```

Comment:
what is this expression supposed to do? 🤔

Reply:
TS will give an error if the schema is not matching with the the original type and vice versa. 
I think it's useful to have since the old type uses types from an external library and the schema does not, so we can use this to make sure they're in sync.